ansible_collections/arista/cvp/plugins/modules/cv_device_v3.py

```diff
@@ -34,7 +34,7 @@
   - Returns number of created and/or deleted containers
 options:
   devices:
-    description: List of devices with their container and configlets information
+    description: List of devices with their container, configlet and image bundle information
```

Comment:
One of the previous commits directly modified the .rst.md file without adding them to the docstrings first and running `make webdoc` as that is the right way to do it. I have updated the change in this PR.